README-zh.md

```diff
@@ -83,6 +84,13 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
   
   默认使用Kmesh功能，可通过调整yaml文件中的启动参数进行功能选择
   
+  ```sh
+  [root@ docker]# vim kmesh.yaml 
+  ...
+             args: ["./start_kmesh.sh -enable-kmesh -enable-ads=true"]

```

Comment:
What's your intention here

Reply:
Here is an example of selecting a function by adjusting the startup parameters in the YAML file.

Reply:
Here is an example of selecting a function by adjusting the startup parameters in the YAML file.

Reply:
This is not an automatical way, in future we can support customize and helm.

No need to provide this in readme

Reply:
OK，Updated